libs/books/data-access/src/lib/+state/reading-list.effects.ts

```diff
@@ -54,9 +55,59 @@ export class ReadingListEffects implements OnInitEffects {
     )
   );
 
+  confirmAddReadingListBook$ = createEffect(() =>
```

Comment:
This logic works for a simple solution, but for purpose of demonstrating good action hygiene, please update this so that "undo" actions for adding/removing books have their own NgRx actions.